storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -62,4 +62,7 @@
     @NonNull
     @CheckResult
     Single<Result> asRxSingle();
+
+    @NonNull
+    Object getData();
```

Comment:
What about providing some common interface or wrapper to avoid passing plane object?